clang/lib/Tooling/DependencyScanning/DependencyScanningWorker.cpp

```diff
@@ -18,6 +18,7 @@
 #include "clang/Tooling/DependencyScanning/ModuleDepCollector.h"
 #include "clang/Tooling/Tooling.h"
 #include "llvm/CAS/CachingOnDiskFileSystem.h"
+#include "llvm/CAS/ObjectStore.h"
```

Comment:
Is this needed?

Reply:
It's because of my change to CASOutputBackend.h to remove HierarchicalTreeBuilder.h (> TreeSchema.h > ObjectStore.h)